piwik/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/continuouspipe/php7-apache
+
+MAINTAINER Alex Biddle <alex.biddle+cp-dockerfiles@inviqa.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y php-geoip php-dev libgeoip-dev && apt-get clean
+
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN echo "extension=geoip.so" >> /etc/php/7.0/apache2/php.ini
+RUN echo "geoip.custom_directory=/app/misc" >> /etc/php/7.0/apache2/php.ini
+
+WORKDIR /app/web
+RUN curl -L -O https://builds.piwik.org/latest.tar.gz && \
+    tar --strip 1 -xzf latest.tar.gz && \
+    rm latest.tar.gz
+
+RUN chown -R www-data:www-data *
```

Comment:
This isn't the safest of operations, are you sure that the web user need to own all of /app/web to function? The safest way to solve this would be to run the untar command above as the `build` user, then reset the permissions for individual web writable files/folders if they must be.